misc-utils/rename.c

```diff
@@ -258,11 +278,19 @@ int main(int argc, char **argv)
 	textdomain(PACKAGE);
 	close_stdout_atexit();
 
-	while ((c = getopt_long(argc, argv, "vsVhnoi", longopts, NULL)) != -1)
+	while ((c = getopt_long(argc, argv, "vsVhnaloi", longopts, NULL)) != -1)
 		switch (c) {
 		case 'n':
 			noact = 1;
 			break;
+		case 'a':
+			all = 1;
+			last = 0;
+			break;
+		case 'l':
+			last = 1;
+			all = 0;
+			break;
 		case 'v':
```

Comment:
It would be better to use _optutils.h_ and mark 'a' and 'l' mutually exclusive. 

See for example misc-utils/lsblk.c with `static const ul_excl_t excl[]` and `err_exclusive_options(c, longopts, excl, excl_st)`. It seems 'o' and 'i' are mutually exclusive too.